core/trino-main/src/main/java/io/trino/server/PluginManager.java

```diff
@@ -146,6 +150,7 @@ private void loadPlugin(String plugin, Supplier<PluginClassLoader> createClassLo
             log.debug("    %s", url.getPath());
         }
 
+        handleResolver.registerClassLoader(pluginClassLoader);
```

Comment:
How does this get unregistered? I only see unregistration for the duplicate one

Reply:
Plugins can not be removed today